examples/CreatePerTitleEncodingDashAndHls.php

```diff
@@ -0,0 +1,398 @@
+
+    $fmp4Muxings = $apiClient->encodings()->muxings($encoding)->fmp4Muxing()->listPage();
+
+    foreach ($fmp4Muxings as $fmp4Muxing) {
+        $segmentPath = $fmp4Muxing->getOutputs()[0]->getOutputPath();
+
+        if (strpos($segmentPath, 'audio') !== false) {
+            # we ignore the audio muxing
+            continue;
+        }
+
+        if (strpos($segmentPath, '{uuid}') !== false) {
+            # we ignore any muxing with placeholders in the path - they are the template muxings, not the result muxings
+            continue;
+        }
+
+        $segmentPath = getSegmentOutputPath(OUTPUT_BASE_PATH, $segmentPath);
+
+        $videoRepresentation_1080p = new DashRepresentation();
+        $videoRepresentation_1080p->setType(DashMuxingType::TYPE_TEMPLATE);
+        $videoRepresentation_1080p->setEncodingId($encoding->getId());
+        $videoRepresentation_1080p->setMuxingId($fmp4Muxing->getId());
+        $videoRepresentation_1080p->setSegmentPath($segmentPath);
+        $apiClient->manifests()->dash()->addRepresentationToAdaptationSet($dashManifest, $period, $videoAdaptationSet, $videoRepresentation_1080p);
+    }
+
+    $apiClient->manifests()->dash()->start($dashManifest);
+
+    // CREATE HLS MANIFEST
+    $hlsManifest = new HlsManifest();
+    $hlsManifest->setName('stream.m3u8');
+    $hlsManifest->setManifestName('stream.m3u8');
+    $hlsManifest->setOutputs(array($manifestOutput));
+    $hlsManifest = $apiClient->manifests()->hls()->create($hlsManifest);
+
+    $mediaInfo = new MediaInfo();
+    $mediaInfo->setGroupId('audio');
+    $mediaInfo->setName('English');
+    $mediaInfo->setUri('audio.m3u8');
+    $mediaInfo->setType(MediaInfoType::AUDIO);
+    $mediaInfo->setSegmentPath('audio/hls/128kbps');
+    $mediaInfo->setMuxingId($audioTSMuxing->getId());
+    $mediaInfo->setStreamId($audioStream->getId());
+    $mediaInfo->setEncodingId($encoding->getId());
+    $mediaInfo->setLanguage('en');
+    $mediaInfo->setAssocLanguage('en');
+    $mediaInfo->setAutoselect(false);
+    $mediaInfo->setDefault(false);
+    $mediaInfo->setForced(false);
+    $mediaInfo = $apiClient->manifests()->hls()->createMediaInfo($hlsManifest, $mediaInfo);
+
+    $tsMuxings = $apiClient->encodings()->muxings($encoding)->tsMuxing()->listPage();
+
+    foreach ($tsMuxings as $tsMuxing) {
+        $segmentPath = $tsMuxing->getOutputs()[0]->getOutputPath();
+
+        if (strpos($segmentPath, 'audio') !== false) {
+            # we ignore the audio muxing
+            continue;
+        }
+
+        if (strpos($segmentPath, '{uuid}') !== false) {
+            # we ignore any muxing with placeholders in the path - they are the template muxings, not the result muxings
+            continue;
+        }
+
+        $segmentPath = getSegmentOutputPath(OUTPUT_BASE_PATH, $segmentPath);
+
+        $streamInfo = new StreamInfo();
+        $streamInfo->setUri('video_' . Uuid::uuid4() . '.m3u8');
```

Comment:
`Uuid` is a package that is only stated as a develop requirement in composer.json and therefore doesn't get installed by default as `composer install --dev` is needed to do so. Therefore it would break the execution of the example, when the API client gets installed as instructed in README. 

Please change the URI naming to something else.